sequencer/src/persistence/fs.rs

```diff
@@ -753,6 +769,16 @@ impl SequencerPersistence for Persistence {
             },
         )
     }
+
+    async fn migrate_consensus(
+        &self,
+        _migrate_leaf: fn(Leaf) -> Leaf2,
+        _migrate_proposal: fn(
+            Proposal<SeqTypes, QuorumProposal<SeqTypes>>,
+        ) -> Proposal<SeqTypes, QuorumProposal2<SeqTypes>>,
+    ) -> anyhow::Result<()> {
+        Ok(())
```

Comment:
Should add a comment on why it's okay to do nothing here and/or link to issue where it will be implemented.